app/backend/src/couchers/migrations/versions/a0d7c8af5402_modify_the_database.py

```diff
@@ -0,0 +1,270 @@
+"""Modify the database
+
+Revision ID: a0d7c8af5402
+Revises: 34cb1d27d207
+Create Date: 2020-12-17 16:43:54.221781
+
+"""
+import geoalchemy2
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a0d7c8af5402"
+down_revision = "34cb1d27d207"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("SET CONSTRAINTS ALL DEFERRED")
```

Comment:
Why?

Reply:
I had to make changes to the migration because of cyclic references. See later comments. But this line is not needed anymore.